broccoli/packages.js

```diff
@@ -0,0 +1,280 @@
+        format: 'es'
+      }]
+    },
+    annotation: 'router.js'
+  });
+}
+
+
+module.exports.jquery = function _jquery() {
+  return new Funnel(findLib('jquery'), {
+    files: ['jquery.js'],
+    destDir: 'jquery',
+    annotation: 'jquery'
+  });
+}
+
+module.exports.internalLoader = function _internalLoader() {
+  return new Funnel('packages/loader/lib', {
+    files: ['index.js'],
+    getDestinationPath() {
+      return 'loader.js';
+    },
+    annotation: 'internal loader'
+  });
+}
+
+module.exports.qunit = function _qunit() {
+  return new Funnel(findLib('qunitjs'), {
+    files: ['qunit.js', 'qunit.css'],
+    destDir: 'qunit',
+    annotation: 'qunit'
+  });
+}
+
+module.exports.glimmerDependencyInjectionES = function _glimmerDependencyInjectionES() {
+  return new Rollup(findLib('@glimmer/di', 'dist/modules/es2017'), {
+    rollup: {
+      entry: 'index.js',
+      dest: '@glimmer/di.js',
+      external: ['@glimmer/util'],
+      format: 'es',
+      exports: 'named'
+    }
+  });
+}
+
+module.exports.emberGlimmerES = function _emberGlimmerES() {
+  return new GlimmerTemplatePrecompiler(new Funnel('packages/ember-glimmer/lib', {
+    include: ['**/*.js', '**/*.hbs'],
+    getDestinationPath(relativePath) {
+      return 'ember-glimmer/' + relativePath;
+    }
+  }), {
+    persist: true,
+    glimmer: require('@glimmer/compiler'),
+    annotation: 'ember-glimmer es'
+  });
+}
+
+module.exports.handlebarsES = function _handlebars() {
+  return new Rollup(findLib('handlebars', 'lib'), {
+    rollup: {
+      entry: 'handlebars/compiler/base.js',
+      dest: 'handlebars.js',
+      format: 'es',
+      exports: 'named',
+      plugins: [handlebarsFix()]
+    },
+    annotation: 'handlebars'
+  })
+}
+
+function handlebarsFix() {
+  var HANDLEBARS_PARSER = /\/parser.js$/;
+  return {
+    load: function(id) {
+      if (HANDLEBARS_PARSER.test(id)) {
+        var code = readFileSync(id, 'utf8');
+        return {
+          code: code
+            .replace('exports.__esModule = true;', '')
+            .replace('exports[\'default\'] = handlebars;', 'export default handlebars;'),
+
+          map: { mappings: null }
+        };
+      }
+    }
+  }
+}
+
+module.exports.rsvpES = function _rsvpES() {
+  let lib = path.resolve(path.dirname(require.resolve('rsvp')), '../lib');
+  return new Rollup(lib, {
+    rollup: {
+      entry: 'rsvp.js',
+      dest: 'rsvp.js',
+      format: 'es',
+      exports: 'named'
+    },
+    annotation: 'rsvp.js'
+  });
+}
+
+module.exports.backburnerES = function _backburnerES() {
+  return funnelLib('backburner.js', 'dist/es6', {
+    files: ['backburner.js'],
+    annotation: 'backburner es'
+  });
+}
+
+module.exports.dagES = function _dagES() {
+  return funnelLib('dag-map', {
+    files: ['dag-map.js'],
+    annotation: 'dag-map es'
+  });
+}
+
+module.exports.routeRecognizerES = function _routeRecognizerES() {
+  return funnelLib('route-recognizer', {
+    files: ['route-recognizer.es.js'],
```

Comment:
We should make an issue and have this lib use the same structure as our other microlibs...